addons/utm/views/utm_stage_views.xml

```diff
@@ -22,6 +22,21 @@
         </field>
     </record>
 
+    <record model="ir.ui.view" id="utm_stage_view_form">
+        <field name="name">utm.stage.view.form</field>
+        <field name="model">utm.stage</field>
+        <field name="priority">10</field>
```

Comment:
Note that default priority is sufficient (16), unless you had reasons to give one ?